packages/website/src/components/channel-banner.module.css

```
@@ -0,0 +1,9 @@
+@reference '../app.css';
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)